src/components/features/Gallery/Gallery.module.scss

```diff
@@ -0,0 +1,308 @@
+@import "../../../styles/settings.scss";
+
+.root {
+  margin-bottom: 2rem;
+}
+
+.leftPanel {
+  margin: 0px 0px;
+  position: relative;
+
+  .heading {
+    position: relative;
+    margin-bottom: 30px;
+    border-bottom: 4px solid $gallery-border;
+
+    h3 {
+      color: $primary;
+      text-transform: uppercase;
+      font-size: 20px;
+      line-height: 38px;
+      margin: 0;
+      letter-spacing: 1px;
+    }
+
+    &::before {
+      content: "";
+      position: absolute;
+      bottom: -4px;
+      left: 0;
+      width: 45px;
+      border-bottom: 4px solid $primary;
+    }
+  }
+
+  .menu {
+    ul {
+      display: flex;
+      padding: 0;
+      margin: 0;
+      justify-content: space-evenly;
+
+      li {
+        list-style-type: none;
+        width: 100%;
+        text-align: center;
+
+        &:last-of-type {
+          a {
+            font-size: 16px;
+            border-right: 1px solid $gallery-border;
+          }
+        }
+      }
+    }
+  }
+
+  .menu a {
+    line-height: 2.5rem;
+    display: block;
+    position: relative;
+    font-weight: 600;
+    border: 1px solid $gallery-border;
+    border-right: none;
+    cursor: pointer;
+    text-transform: uppercase;
+
```

Comment:
jakbyś tu dał właściwość text-decoration:none; to zniknęłoby to podkreślenie które jest domyślne w a

Reply:
Basia, nie widzę miejsca gdzie w tym komponencie pojawiają sie podkreślenia.

Reply:
jak najeżdżasz na TOP SELLER to pojawia się dolne podkreślenie

Reply:
Basia dziękuję. Nie zauważyłem wcześniej.